test/ch/query_test.exs

```diff
@@ -113,7 +113,7 @@ defmodule Ch.QueryTest do
     test "decoded binaries copy behaviour", %{conn: conn} do
       text = "hello world"
       assert [[bin]] = Ch.query!(conn, "SELECT {$0:String}", [text]).rows
-      assert :binary.referenced_byte_size(bin) == byte_size(text)
+      assert :binary.referenced_byte_size(bin) == :binary.referenced_byte_size("hello world")
```

Comment:
Seems like OTP-27 over-allocates memory for small binaries, so this test had to be updated since references byte size for "hello world" is 88 bytes instead of 11